### PR TITLE
changed the header to be Pascal on CommuteDistanceDistribution component

### PR DESCRIPTION
**Pull request by FaneTree**
## Overview of the Pull Request:
_Describe your changes here..._

## How Has This Been Tested?
- Please describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.  => Change only lines 26 and 40 about "Distribution of Commute Distance" on CommuteDistanceDistribution component
- If necessary, please also list any relevant details for your test setup/configuration

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ ] Have you written instructions on how to test that your changes work as expected?
- [ ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by OlaShabalina** [comment]
Hi @FaneTree , congrats on first commit! 

What about the following charts?
image

Also, would you be able to revert the changes to `package.json` and `package-lock.json`, those would need to be done in a separate PRs to keep the code single purposed.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "framer-motion": "^10",
     "highcharts": "^10.3.3",
     "highcharts-react-official": "^3.2.0",
-    "next": "^13.2.3",
+    "next": "^13.3.0",
```
Comment (OlaShabalina) [inline package.json]:
```suggestion
    "next": "^13.2.3",
```

**Review by OlaShabalina — CHANGES_REQUESTED** [review]

**Review by Amutha37 — APPROVED** [review]
Great first commit Fane.

**Review by OlaShabalina — CHANGES_REQUESTED** [review]
Could you please revert changes to `package-lock.json` as well? Just the code that is needed for this task.

**Comment by FaneTree** [comment]
I have changed all as discussed. To summarise all the changes.
- Adjusted in CommuteDistanceDistribution component
    - line 26 (For header before description),
    - line 40 (for chart header)
- Adjusted in TripCountAndTravelMethods component
    - line 44 Trip Count and Travel Methods ⇒ Count
    - line 69 "Active-public-shared Travel” ⇒ Travel
    - line 81 "Individual Travel” ⇒ Travel

Note: 
1. checked <Text> CSS and both are the same for “Travel to Work Days” and “Distribution of Commute Distance”.
2. pull the current version again to align the version and now the local and this repository are the same.

**Comment by OlaShabalina** [comment]
Missed 1:
image

**Comment by hqtan** [comment]
Awesome work on your first PR to this repo @FaneTree ! 🙌🏽 

You're _almost_ done with this PR; remember to address @OlaShabalina 's comment about removing `package*.json` files from this PR k?
Please sing out if you need any assistance!

🙇🏽

**Review by OlaShabalina — APPROVED** [review]
Well done @FaneTree , looks good to me! 👍🏻

**Pull request merged**